src/AltinnCore/Designer/TypedHttpClients/AzureDevOps/AzureDevOpsBuildService.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using AltinnCore.Common.Configuration;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps.Models;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace AltinnCore.Designer.TypedHttpClients.AzureDevOps
+{
+    /// <summary>
+    /// Implementation of IAzureDevOpsService
+    /// </summary>
+    public class AzureDevOpsBuildService : IAzureDevOpsBuildService
+    {
+        private readonly HttpClient _httpClient;
+        private readonly GeneralSettings _generalSettings;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="httpClient">System.Net.Http.HttpClient</param>
+        /// <param name="generalSettingsOptions">IOptionsMonitor of Type GeneralSettings</param>
+        public AzureDevOpsBuildService(
+            HttpClient httpClient,
+            IOptionsMonitor<GeneralSettings> generalSettingsOptions)
+        {
+            _generalSettings = generalSettingsOptions.CurrentValue;
+            _httpClient = httpClient;
+        }
+
+        /// <inheritdoc/>
+        public async Task<Build> QueueAsync(
+            QueueBuildParameters queueBuildParameters,
+            int buildDefinitionId)
+        {
+            queueBuildParameters.GiteaEnvironment = (Environment.GetEnvironmentVariable("GeneralSettings__HostName") ?? _generalSettings.HostName) + "/repos";
```

Comment:
Did I catch you wrong during daily or could this be simplified after @altinnadmin's fix ?

Reply:
You are correct, that can be simplified and will be simplified now.